nodes/src/nodes/memory_persistent/memory_store.py

```
@@ -0,0 +1,613 @@
+        summary_entry = {
+            'op': 'summary',
+            'summarized_count': len(old_entries),
+            'op_counts': op_counts,
+            'keys_touched': sorted(keys_touched),
+            'timestamp': time.time(),
+        }
+
+        # Replace history: summary + recent entries
+        if isinstance(self._backend, InMemoryBackend):
+            with self._backend._lock:
+                self._backend._history[session_id] = [summary_entry] + recent_entries
```

Ruff (0.15.7)

[warning] Consider `[summary_entry, *recent_entries]` instead of concatenation

Replace with `[summary_entry, *recent_entries]`

(RUF005)